### PR TITLE
README: add install/build notes

### PR DESCRIPTION
**Pull request by landam**
Update readme:

* how to install hugo (Debian)
* how to build website

### CODE REVIEW

**Comment by lucadelu** [comment]
I tried running hugo and it seems to work, but the output has some problem with link, some are redirecting to @nbozon website other are local but broken (like file:///GRASS-GIS/news/new-crowdfunding-campain/)

File: README.md [hunk 1]
```diff
@@ -1,2 +1,17 @@
...
+
+    apt install hugo
+
+## How to build
+
+Run in the `grass-website` root directory:
+
+    hugo
+
+Open web site (using Mozilla Firefox in this case):
```
Comment (lucadelu) [inline README.md]: Before this I think is better to link public folder it to a web server directory (as suggested by @nbozon), so the links are working better (not yet correctly)
```suggestion
Link the public folder to your web server directory

ln -s `pwd`/public /your/web/server/directory/grass-website
```

File: README.md [hunk 2]
```diff
@@ -1,2 +1,17 @@
...
+### Debian
+
+    apt install hugo
+
+## How to build
+
+Run in the `grass-website` root directory:
+
+    hugo
+
```
Comment (lucadelu) [inline README.md]: Before this I think is better to link public folder it to a web server directory (as suggested by @nbozon), so the links are working better (not yet correctly)
```suggestion
Link the public folder to your web server directory

ln -s `pwd`/public /your/web/server/directory/grass-website
```

File: README.md [hunk 3]
```diff
@@ -1,2 +1,17 @@
...
+Open web site (using Mozilla Firefox in this case):
+
+    firefox public/index.html
```
Comment (lucadelu) [inline README.md]: If previous suggestion will be accepted this line should be changed as

```suggestion
    firefox http://localhost/grass-website
```

Comment (landam) [reply README.md]: I update how to run hugo server locally. @lucadelu Feel free to open new PR for running website on production server.

**Pull request merged**